### ISSUE TITLE
SkipTags support  regular expression

### ISSUE THREAD

**Issue by wuseal** [joffrey-bion/gradle-github-changelog#36]
Very Glad to find this Repo, After trying it, I found something need enhancement I think, If not support, I will need to list all of the tags that should not be show in Change Log like this 
```kotlin
    skipTags = listOf(
        "v2.1-Alpha",
        "2.1-EAP",
        "EAP",
        "2.2.0.1-EAP",
        "2.2.1-EAP",
        "2.2.2-EAP",
        "2.2.3-EAP",
        "2.2.4-EAP",
        "V3.0-EAP",
        "V3.0-EAP-2",
        "v3.2.0-EAP",
        "3.3.0-EAP",
        "3.3.0-EAP-2",
        "3.4.0-EAP",
        "3.4.0-EAP-2",
        "3.5.0-EAP",
        "3.5.1-EAP",
        "3.6.0-EAP",
        "3.5.0-EAP-2",
        "3.7.0-EAP",
        "3.7.0-EAP-2",
        "3.7.0-EAP-3"
    )
```
It looks badly

I hope it could be set like this:
```kotlin 
skipTags = listOf(".*EAP.*", ".*Alpha.*")
```

**Comment by joffrey-bion** [comment]
This is a good point, thanks! I had no need for regex so far, but it might be useful in some cases.
Will give it a try when I have some time.

**Comment by joffrey-bion** [comment]
Released in 1.7.0

**Comment by wuseal** [comment]
Very happy to see it was implemented. Nice property : skipTagsRegex. Thanks man
